### PR TITLE
First version of vectorCalculator

### PR DESCRIPTION
**Pull request by d99joper**
First attempt at a gephi plugin, and first time creating a pull request. Let me know if I'm doing this the right way.

Jonas

### CODE REVIEW

File: modules/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Gephi is an interactive visualization and exploration platform for all kinds of networks and complex systems, dynamic and hierarchical graphs.
```
Comment (eduramiba) [inline modules/README.md]: This file should be removed before merge

File: modules/VectorCalculator/pom.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>nbm-maven-plugin</artifactId>
+                <configuration>
+                    <licenseName>Apache 2.0</licenseName>
+                    <author>Jonas Persson</author>
+                    <authorEmail></authorEmail>
+                    <authorUrl></authorUrl>
+                    <sourceCodeUrl>$sourcecode_url</sourceCodeUrl>
```
Comment (eduramiba) [inline modules/VectorCalculator/pom.xml]: This is a bug in the plugin creator, please fill in the correct URL

File: modules/README.md (hunk 1 above)
Comment (d99joper) [inline modules/README.md]: I deleted the file

File: modules/VectorCalculator/pom.xml (hunk 2 above)
Comment (d99joper) [inline modules/VectorCalculator/pom.xml]: This is completed

**Comment by eduramiba** [comment]
You should also add the modules/VectorCalculator line to the root pom.xml file

**Comment by d99joper** [comment]
Not sure if I know what you mean by "_You should also add the modules/VectorCalculator line to the root pom.xml file"_, but I modified the sourceUrl again. Let me know if you meant something else.

**Comment by eduramiba** [comment]
I mean this file https://github.com/gephi/gephi-plugins/blob/master-forge/pom.xml should contain the module, but since the merge has to be manual I could add it anyway.

**Comment by d99joper** [comment]
Got it- I added it to my root pom.xml

**Comment by eduramiba** [comment]
Thanks! Will check to merge soon

**Comment by eduramiba** [comment]
Merged in commit 644747acf3d7dc37b87d4e8386a128895cb7f405

**Comment by eduramiba** [comment]
Your plugin is now available at the update center :smiley:

**Pull request closed without merging**